### PR TITLE
Render edit page passing attributes to the branch form object

### PR DESCRIPTION
**Pull request by tomas-stefano**
[Trello card](https://trello.com/c/vRkJTidU/1740-saving-edit-branch-page-after-editing-updating)

## Context

This PR adds the branch edit/update page.

The edit will render the form loading from the metadata and then saving by replacing the flow object with the new parameters.

### CODE REVIEW

File: app/views/services/edit.html.erb [hunk 1]
```diff
@@ -35,6 +35,10 @@
               method: :delete, class: 'destructive delete' %>
             <% end %>
           </li>
+          <li data-action="edit">
+            <%= link_to 'Add branching',
+              new_branches_path(service.service_id, page.uuid) %>
+          </li>
```
Comment (brenetic) [inline app/views/services/edit.html.erb:41]: can we take this out please. it's on the `testable-add-branching-link` branch

File: config/routes.rb [hunk 2]
```diff
@@ -18,9 +18,9 @@
     member do
       resources :publish, only: [:index, :create]
       resources :pages, param: :page_uuid, only: [:create, :edit, :update, :destroy]
-      resources :branches, param: :previous_flow_uuid, only: [:create, :edit] do
-        member do
-          get '/new', to: 'branches#new'
+      resources :branches, param: :branch_id, only: [:create, :edit, :update] do
```
Comment (brenetic) [inline config/routes.rb:21]: `branch_uuid`? 🤷  just a thought

Comment (tomas-stefano) [reply config/routes.rb]: Done

File: app/views/services/edit.html.erb (hunk 1 above)
Comment (tomas-stefano) [reply app/views/services/edit.html.erb]: Done

**Review by brenetic — APPROVED** [review]
🚀

**Pull request merged**